supplements/journeys-through-the-radiant-citadel/items.xml

```diff
@@ -0,0 +1,93 @@
+		</description>
+		<setters>
+			<set name="category">Adventuring Gear</set>
+			<set name="cost" currency="gp">2</set>
+			<set name="weight" lb="2">2 lb.</set>
+			<set name="stackable">true</set>
+		</setters>
+	</element>
+
+	<element name="White Ghost Orchid Seed" type="Item" source="Journeys Through the Radiant Citadel" id="ID_WOTC_JTRC_WHITE_GHOST_ORCHID_SEED">
+		<description>
+			<p>More rarely, the orchids produce a smaller pod holding a single white seed. Among its various magical properties, if a white seed is ground and scattered over a dead body, the body is affected by the <i>resurrection</i> spell. A white ghost orchid seed has no effect if consumed.</p>
+		</description>
+		<setters>
+			<set name="category">Adventuring Gear</set>
+			<set name="cost" currency="gp">2</set>
+			<set name="weight" lb="2">2 lb.</set>
+			<set name="stackable">true</set>
+		</setters>
+	</element>
+
+	<element name="Hammock of Worlds" type="Magic Item" source="Journeys Through the Radiant Citadel" id="ID_WOTC_JTRC_MAGIC_ITEM_HAMMOCK_OF_WORLDS">
+		<description>
+			<p>The Hammock of Worlds is a colorful hammock woven with traditional Ataguan designs. It can be used only by a member of the order known as the Green Doctors.</p>
+			<p class="indent">You can use an action to unfold and place the Hammock of Worlds on a solid surface, whereupon it creates a two-way portal to the Ghost Orchid Tepui in the Feywild or to the Drought Elder in the Far Realm (the user's choice). You can use an action to close an open portal by taking hold of the edges of the cloth and folding it up. Once Hammock of Worlds has opened a portal, it can't do so again for 1d8 hours.</p>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Legendary</set>
+		</setters>
+	</element>
+
+	<element name="Lesser Hammock of Worlds" type="Magic Item" source="Journeys Through the Radiant Citadel" id="ID_WOTC_JTRC_MAGIC_ITEM_LESSER_HAMMOCK_OF_WORLDS">
+		<description>
+			<p>The Hammock of Worlds is a colorful hammock woven with traditional Ataguan designs.</p>
+			<p class="indent">You can use an action to unfold and place the lesser version of the Hammock of Worlds on a solid surface, whereupon it creates a two-way portal to another world or plane of existence. Each time the item opens a portal, the user can cause it to always reliably connect to the Ghost Orchid Tepui. You can use an action to close an open portal by taking hold of the edges of the cloth and folding it up. Once the lesser version of the Hammock of Worlds has opened a portal, it can't do so again for 1d8 hours.</p>
```

Comment:
i believe this is how it should actually be, based on the wording of normal Hammock.
```suggestion
			<p class="indent">You can use an action to unfold and place the lesser version of the Hammock of Worlds on a solid surface, whereupon it creates a two-way portal to the Ghost Orchid Tepui in the Feywild. You can use an action to close an open portal by taking hold of the edges of the cloth and folding it up. Once the lesser version of the Hammock of Worlds has opened a portal, it can't do so again for 1d8 hours.</p>
```